### DISCUSSION TITLE
Request to implement Grafana dashboard mixin using jsonnet

### DISCUSSION THREAD

**Discussion by thanatchakromsang** [fluxcd/flux2#1750] [Proposals]
I have already implemented Jsonnet implementation for `flux-control-plane` and `flux-status` dashboards  for my own usage but I think It will benefit more If got merged into flux v2 repo for other people to extends and override selector or even do multi cluster dashboards as well 

Would you guy be okay If I open a pull request for the mixin?

**Comment by nlamirault** [comment]
It would be nice to initialize a fluxcd mixin ?

**Reply by thanatchakromsang** [reply]
Sound good to me, but I do not think the mixin is big enough to justify having a it own repository

**Reply by nlamirault** [reply]
@thanatchakromsang yes. Just a mixin subdirectory

**Reply by thanatchakromsang** [reply]
Ok sure, I will create a mixin PR later this weekend

**Reply by nlamirault** [reply]
@thanatchakromsang any news ?

**Reply by stefanprodan** [reply]
A Flux mixin could he hosted in the [fluxcd-community](https://github.com/fluxcd-community) org. Let me know if i should create a repo there. If this takes off, we could deprecate the dashboards from flux2 repo and point users to the mixin repo.

**Reply by thanatchakromsang** [reply]
@stefanprodan It would be good If you could create the repository there. I will send a PR. thanks!

**Reply by stefanprodan** [reply]
Here it is: https://github.com/fluxcd-community/flux-mixin

Thanks!

**Reply by nlamirault** [reply]
Great !

**Reply by stefanprodan** [reply]
@thanatchakromsang @nlamirault I can give you maintainer access to this repo, if you are willing to drive this.

**Reply by thanatchakromsang** [reply]
Please do. thanks!

**Reply by stefanprodan** [reply]
@thanatchakromsang check your email you should have an invite. Thanks!

**Reply by nlamirault** [reply]
@stefanprodan why not ... Thanks.

**Reply by stefanprodan** [reply]
@nlamirault invitation sent

**Reply by thanatchakromsang** [reply]
@nlamirault @stefanprodan I have dashboard mixin ready, just need to modify a bit for flux mixin. I will have the pull request ready tomorrow

**Reply by thanatchakromsang** [reply]
The pull request is ready https://github.com/fluxcd-community/flux-mixin/pull/1

**Reply by stefanprodan** [reply]
@nlamirault are you able to help test and review the PR?

**Reply by nlamirault** [reply]
@stefanprodan No :( I don't have any cluster with Flux ready for now ...

**Reply by nlamirault** [reply]
I make a comment into the PR.
